src/app/core/core.module.ts

```diff
@@ -188,6 +214,42 @@ const PROVIDERS = [
   { provide: NativeWindowService, useFactory: NativeWindowFactory }
 ];
 
+/**
+ * Declaration needed to make sure all decorator functions are called in time
+ */
+export const normalizedModels =
+  [
+    NormalizedDSpaceObject,
+    NormalizedBundle,
+    NormalizedBitstream,
+    NormalizedBitstreamFormat,
+    NormalizedItem,
+    NormalizedCollection,
+    NormalizedCommunity,
+    NormalizedEPerson,
+    NormalizedGroup,
+    NormalizedResourcePolicy,
+    NormalizedMetadataSchema,
+    NormalizedMetadataField,
+    NormalizedLicense,
+    NormalizedWorkflowItem,
+    NormalizedWorkspaceItem,
+    NormalizedSubmissionDefinitionsModel,
+    NormalizedSubmissionFormsModel,
+    NormalizedSubmissionSectionModel,
+    NormalizedSubmissionUploadsModel,
+    NormalizedAuthStatus,
+    NormalizedAuthorityValue,
+    NormalizedBrowseEntry,
+    BrowseDefinition,
```

Comment:
This looks out of place.  Should this be listed in the `normalizedModels` section?  Just curious why it's the only one not starting with `Normalized` in this list.

Reply:
Because there is no NormalizedBrowseDefinition model. 

BrowseDefinition is used as both the rest-side and client-side model. Creating one, and splitting up the usages is not trivial, so I felt it was out of scope for this task.

I created an issue to track it: https://github.com/DSpace/dspace-angular/issues/436

Reply:
Thanks for the explanation. That sounds like a good resolution to me, and I agree it's not high priority enough to put into this PR.